workflows/genome_annotation/functional-annotation/functional-annotation-protein-sequences/.dockstore.yml

```
@@ -0,0 +1,15 @@
+version: 1.2
+workflows:
+- name: main
+  subclass: Galaxy
+  publish: true
+  primaryDescriptorPath: /Functional_annotation_of_protein_sequences.ga
+  testParameterFiles:
+  - /Functional_annotation_of_protein_sequences-tests.yml
+  authors:
+  - name: Romane Libouban
+    email: mailto:romane.libouban@irisa.fr
+    orcid: 0009-0001-4920-9951
+  - name: Anthony Bretaudeau
+    email: mailto:anthony.bret@irisa.fr
```

```suggestion
    email: mailto:anthony.bretaudeau@irisa.fr
```
was it autogenerated or just a copy/paste accident?

---

autogenerated from the workflow file, I'm not about copy-paste :)